### PR TITLE
fix for #483

### PR DESCRIPTION
**Pull request by i-make-robots**
adds parsing for `M280 Snn` where nn is the servo angle.

### CODE REVIEW

File: src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCode.java [hunk 1]
```diff
@@ -73,100 +82,104 @@ public Turtle load(InputStream in) throws Exception {
 						penDownColor = new ColorRGB(Integer.parseInt(color.substring(1)));
 						turtle.setColor(penDownColor);
 						break;
+					case 280:
+						// z move
+						double v = Double.valueOf(tokenExists("S",tokens));
```
Comment (PPAC37) [inline src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCode.java:87]: double v = Double.valueOf(tokenExists("S",tokens).substring(1));

**Comment by PPAC37** [comment]
Well done I was completely focused on G0 vs G1 whereas the modification you propose works perfectly after the correction of the substring.

On the other hand, while testing, I noticed that we don't read the color change / tool change ... that's another story and I don't even know where to start if we don't guarantee that the M0 will not always have the same message format and will always be an M0 for the color change.

File: src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCode.java (hunk 1 above)
Comment (i-make-robots) [reply src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCode.java:87]: DOH

**Comment by i-make-robots** [comment]
M6 is the command for a tool change, M0 is only a message.  I don't remember if we do anything with M6 or how Marlin handles it.

**Comment by PPAC37** [comment]
> M6 is the command for a tool change, M0 is only a message. I don't remember if we do anything with M6 or how Marlin handles it.

Sorry for the mixing Turtle contexte (TurtleMove.TOOL_CHANGE) and marlin gcode equivalent for me 
M280 ... ; pen Up 
M0 please change color to ... 
M280 ... ; Pen down 

( No M6 in marlin g-codes https://marlinfw.org/meta/gcode/ (maybe on repetier firmware or else ) but T...  https://marlinfw.org/docs/gcode/T001-T002.html for selecting tools on marlin. )

**Pull request merged**